docs/releases/release-1.0.md

```diff
@@ -0,0 +1,56 @@
+# Kubeflow Pipelines 1.0 Release Notes
+
+## Versioning policy
+
+Kubeflow Pipelines 1.0 is the first major release of Kubeflow Pipelines. The
+Kubeflow Pipelines versioning follows the conventional versioning policy. That
+is, the major version of Kubeflow Pipelines is incremented for API/feature changes that
+are new and not backward-compatible; and on the other hand, the minor version
+of Kubeflow Pipelines is incremented for API/feature changes that are new and
+backward-compatible.
+
+## Deployment options
```

Comment:
I am thinking instead of mentioning infeasible approaches, i just mention feasible approaches, hence these two deployments. And we'll probably won't have kubeflow soon, only have kubeflow pipelines....

Reply:
done